include/ibamr/INSVCStaggeredConservativeHierarchyIntegrator.h

```diff
@@ -169,6 +172,20 @@ class INSVCStaggeredConservativeHierarchyIntegrator : public INSVCStaggeredHiera
      */
     void registerMassDensitySourceTerm(SAMRAI::tbox::Pointer<IBTK::CartGridFunction> S_fcn);
 
+    /*!
+     * Returns the number of cycles to perform for the present time step.
+     */
+    int getNumberOfCycles() const;
+
+    /*!
+     * Get the convective operator being used by the integrator class.
+     *
+     * \note The class employs INSVCStaggeredConservativeMassMomentum integrator
+     * to compute convective derivative. Therefore, ConvectiveOperator is a NULL
+     * object.
+     */
+    SAMRAI::tbox::Pointer<ConvectiveOperator> getConvectiveOperator();
```

Comment:
I am a little confused by the comment here; if this is always NULL, do we need this method?

Reply:
Yep😎. Since we are not using any convective op now, we set it to NULL. No point in having a default convective op hanging around and waste initialization compute time.

Reply:
Maybe this method eventually needs to be pushed down into a constant-coefficient subclass of the parent class...

Reply:
We actually use a convective op for non-conservative form. Density is multiplied to (u. grad u) term obtained from convective op.